### ISSUE TITLE
Sorting does not work

### ISSUE THREAD

**Issue by Ollo1** [Alcidauk/CineLog#5]
Cinelog v1.0
Android 8.0

Changing the order of movies does not work for Year and Date Added. Neither in ascending nor descending order.

**Comment by Alcidauk** [comment]
Thats true I reproduce it. It's a part I did not rewrite yet. Will try to fix this quickly, thanks for this report !
